src/Discovery/aggMDSUtils.jsx

```diff
@@ -91,9 +92,7 @@ const loadStudiesFromAggMDSRequests = async (offset, limit) => {
           }
         }
       });
-
-      x.tag = getUniqueTags(x.tags);
-
+      x.tags = [...getUniqueTags(x.tags).entries()].map((e) => (e[1]));
```

Comment:
I have to say I don't really get this: if `getUniqueTags()` still can have dups in return value, why don't change that function to make it work more appropriately?

And why the old code is updating `x.tag` while this is updating `x.tags`?

Reply:
Let me refactor and put it within getUniqueTags(). x.tag is never used anywhere. I assumed the intention was x.tags

Reply:
I have refactored getUniqueTags()